<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupWidget</class>
 <widget class="QWidget" name="SetupWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请选择游戏参数</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <layout class="QGridLayout" name="layoutGWidget">
     <item row="5" column="0" colspan="2">
      <layout class="QHBoxLayout" name="layoutHBtn">
       <item>
        <widget class="QPushButton" name="btnJoin">
         <property name="text">
          <string>加入牌桌</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCance">
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelRaiseType">
       <property name="text">
        <string>加注类型</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <spacer name="spacerVM">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtPlayerCnt">
       <property name="text">
        <string>3</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelBB">
       <property name="text">
        <string>大盲注</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtInitChip">
       <property name="text">
        <string>1000</string>
       </property>
       <property name="placeholderText">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelInitChip">
       <property name="text">
        <string>初始筹码</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelPlayerCnt">
       <property name="text">
        <string>用户数</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxRaiseType">
       <item>
        <property name="text">
         <string>有限加注</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>无限加注</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtBB">
       <property name="text">
        <string>4</string>
       </property>
       <property name="placeholderText">
        <string>4</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="spacerVB">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="spacerHL">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>31</width>
       <height>188</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="spacerHR">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>188</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
